java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1196,7 +1225,7 @@ public static HostColumnVector timestampNanoSecondsFromBoxedLongs(Long... values
   /**
    * WARNING: Debug only method to print a passed in buffer
    */
-  private static void printBuffer(HostMemoryBuffer buffer) {
+  static void printBuffer(HostMemoryBuffer buffer) {
```

Comment:
Does this need to be public in the final version?

Reply:
Yes, making it so will be useful.

Reply:
The like I said when this was added originally if this is going to be a public API it should move to HostMemoryBuffer.

Reply:
Done.